apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/CharSequenceContainsAtLeastAssertionsSpec.kt

```diff
@@ -69,7 +73,7 @@ class CharSequenceContainsAtLeastAssertionsSpec : ch.tutteli.atrium.spec.integra
         )
 
         private fun containsAtLeastButAtMostIgnoringCase(plant: Assert<CharSequence>, atLeast: Int, butAtMost: Int, a: Any, aX: Array<out Any>)
-            = plant to contain ignoring case atLeast atLeast butAtMost butAtMost the Values(a, *aX)
+            = (plant.asExpect().contains(o) ignoring case atLeast atLeast butAtMost butAtMost).the(Values(a, *aX))
```

Comment:
getErrorMsgContainsNot is one example where we need to adjust the error message. There are two things to do;
- change it from `use $containsNotValues instead of `$atLeast $times` to `use `$containsNotValues` instead of `$atLeast $times`
- change containsNotValues in CharSequenceContainsSpecBase to `"${containsNotFun.name} values`

Just that you know it already, similar things need to be done for iterable once you tackle them